### ISSUE TITLE
Test Suite

### ISSUE THREAD

**Issue by DavidBM** [rzip/rzip#14]
I'm trying to find a test suite for unzip, but seems that they don't have any?

Do you know of any test suite for zips?

**Comment by DavidBM** [comment]
https://github.com/rzip/zip-rs/tree/master/tests/data
We have this files to test. I don't see zip-rs using them, therefore, we can add them to the rzip repo and use them as edge cases tests.

**Comment by elpiel** [comment]
I guess we can test the binary with files like that, but I don't think we need to test edge-cases that `zip-rs` should cover as the underlying library we are using.

**Comment by DavidBM** [comment]
I would agree if zip-rs was maintained and properly tested, but I don't see full coverage in the library. I think in our case we can add some tests just in case. Later we can work a better solution, but, for example, if we find a test suite, maybe it is worth it to run rzip under it.
